test/Atmos/EDMF/bomex_edmf.jl

```diff
@@ -4,6 +4,7 @@ ClimateMachine.init(;
     output_dir = get(ENV, "CLIMATEMACHINE_SETTINGS_OUTPUT_DIR", "output"),
     fix_rng_seed = true,
 )
+@show ClimateMachine.array_type()
```

Comment:
Is this needed now in all edmf experiment files?

Reply:
It's not needed, but I find it convenient to know whether it's running CPU/GPU right away.